dids/index.js

```diff
@@ -1,10 +1,11 @@
 /*!
  * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
  */
-'use strict';
-
+import {createRequire} from 'node:module';
+const require = createRequire(import.meta.url);
 const requireDir = require('require-dir');
```

Comment:
we might want to look into replacing `require-dir` in the future. I'll make an issue about it.